### ISSUE TITLE
Flows not being saved.

### ISSUE THREAD

**Issue by bobcroft** [sakazuki/node-red-desktop#29]
If I import a flow I can deploy it successfully, I can add nodes, in fact everything appears normal.  I click save to save the flow.  If I then shut down Node-RED Desktop and then re-start it there are no flows.  I have tried this with one flow and two flows, everything is fine until I close Node-RED Desktop down, then when I reopen it there are no flows at all.  Can someone please tell me what I am doing wrong.  PC is Windows 10 * 64  and I used the NRD installer to install it.

**Comment by sakazuki** [comment]
Please confirm the settings.
Do you check [File]-[Settings...]-[Other]-[Open a last file on startup] ?

**Comment by bobcroft** [comment]
Hi,

 

Thank you for your reply.  Yes I did check [File]-[Settings...]-[Other]-[Open a last file on startup] and it is ticked.  Just to be sure I re-started the program and still there are no flows.  I am running the latest version of Windows 10 Pro.

 

Thanks

 

Bob

**Comment by bobcroft** [comment]
just tried again this time with a new flow and just two nodes.  I close Node-RED Desktop, re-open it no flows!  It seems fairly elementary that it should save flows to me.  Have you any further suggestions please.

**Comment by sakazuki** [comment]
Please check a log from [File]-[Open log file].
You will find a line of "Flows file" like the following
```
[2020-10-25 05:45:03.062] [info] Flows file     : C:\Users\XXXX\.Node-RED-Desktop\Node-RED-Desktop--6896-vckTCzgXZ09u-.tmp
```
Let me see the outputs

**Comment by bobcroft** [comment]
Hi,

 

I needed to get Node-RED working so I uninstalled Node-RED desktop and installed Node-RED in the conventional way as per the node-RED documents and it now works perfectly.  However, I really liked your desktop version so I’ll install it on another PC and try it again.  If that fails I’ll send you the log you requested.  If it works OK on my second PC I’ll also try it again on my main PC as it seems the two instances use a different folder structures.

 

Bob
